src/lib/tooltip/tooltip.test.ts

```diff
@@ -434,6 +434,17 @@ describe('Tooltip', () => {
       expect(harness.isOpen).to.be.true;
     });
 
+    it('should open when hovering the trigger button and a delay is set', async () => {
+     const harness = await createFixture({ triggerType: 'hover', delay: 500 });
```

Comment:
Did you intend to add this test to the `popover.test.ts` file instead?

Reply:
I was referencing the tooltip file and accidentally added it there. I can remove it from there and move it over to the popover